frontend/src/redux/slices/position-slice.ts

```diff
@@ -51,13 +53,31 @@ export const positionSlice = createAppSlice({
         },
       },
     ),
+    fetchPosition: create.asyncThunk(
+      async (id: string) => {
+        return await getPositionById(id);
```

Comment:
We should only fetch if this data isn't already available in the state

Reply:
So there should be a new field in the state that holds already fetched positions. 

Though we could skip implementing this currently, because usage of RTQ query would bring support this by design (if we will start using RTK query)

Reply:
Yeah, mäybe time for refactoring later on